install_libcxx.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-TRUNK_VERSION="13.0.1"
+TRUNK_VERSION="14.0.1"
```

Comment:
Might want to have a closer look at this value.  By my reckoning it should be "15.0.0".

Reply:
Changing to `15.0.0` fails Clang 14 builds since they try downloading from non-existing releases.

Reply:
My bad.